src/main/app.js

```diff
@@ -16,6 +16,11 @@ class App {
       app.commandLine.appendSwitch('enable-experimental-web-platform-features', 'true')
     }
 
+    // Enable vscode chrome extension debugger connection
+    if (process.env.NODE_ENV !== 'production' || process.env.NODE_ENV === 'development') {
```

Comment:
Personally I would check for `process.env.NODE_ENV === 'development'` only. First of all because of security reasons and it's redundant.